src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -500,6 +500,13 @@ def load_arguments(self, _):
 
         c.argument('protect_from_scale_in', arg_type=protection_policy_type, help="Protect the VM instance from scale-in operations.")
         c.argument('protect_from_scale_set_actions', arg_type=protection_policy_type, help="Protect the VM instance from scale set actions (including scale-in).")
+        c.argument('enable_terminate_notification', min_api='2019-03-01', arg_type=get_three_state_flag(),
+                   help='Enable terminate notification')
+
+    for scope in ['vmss create', 'vmss update']:
+        with self.argument_context(scope) as c:
+            c.argument('terminate_notification_time', min_api='2019-03-01',
+                       help='Length of time (in minutes) a notification to be sent to the VM on the instance metadata server till the VM gets deleted')
```

Comment:
I prefer adding the limitation information into help without checking it.

Reply:
> 
> 
> I prefer adding the limitation information into help without checking it.

Good idea!